src/main/java/org/opentripplanner/middleware/otp/OtpRequestProcessor.java

```diff
@@ -13,81 +13,121 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import spark.Request;
+import spark.Service;
 
 import static org.opentripplanner.middleware.auth.Auth0Connection.isAuthHeaderPresent;
 import static org.opentripplanner.middleware.spark.Main.getConfigPropertyAsText;
 import static org.opentripplanner.middleware.utils.JsonUtils.logMessageAndHalt;
 
 /**
- * Responsible for getting a plan response from OpenTripPlanner based on the parameters provided by requester. If the
- * user is known and they have given consent store the trip. Pass back to the requester the original response and HTTP
- * status code provided by OpenTripPlanner.
+ * Responsible for getting a response from OTP based on the parameters provided by the requester. If the target service
+ * is of interest the response is intercepted and processed. In all cases, the response from OTP (content and HTTP status)
```

Comment:
Some additional context would be helpful in this javadoc. Also, probably needs additional changes related to its role in proxying all requests:

> Responsible for proxying any and all requests made to its HTTP endpoint to OTP. If the target service is of interest (e.g., requests made to the plan trip endpoint are currently logged if the user has consented to storing trip history) the response is intercepted and processed. In all cases, the response from OTP (content and HTTP status) is passed back to the requester.

Reply:
Updated.